ui/packages/components/src/Timeline/TimelineNode/TimelineNode.tsx

```diff
@@ -17,11 +17,13 @@ import { renderTimelineNode } from './TimelineNodeRenderer';
 type Props = {
   getOutput: (historyItemID: string) => Promise<string | undefined>;
   node: HistoryNode;
-  position: 'first' | 'last' | 'middle';
+  position?: 'first' | 'last' | 'middle';
+  children?: React.ReactNode;
+  type?: 'attempt';
```

Comment:
I'm not sure if this would be better as a boolean like isAttempt

Reply:
If there are just 2 states then a boolean would be good